test/e2e/backup_test.go

```diff
@@ -0,0 +1,83 @@
+package e2e
+
+import (
+	"flag"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+var (
+	backupName  string
+	restoreName string
+)
+
+var _ = Describe("Backup Restore test using Kibishii to generate/verify data", func() {
+
+	BeforeEach(func() {
+		flag.Parse()
+	})
+	Describe("backing up and restoring namespace with data", func() {
+		Context("when the backup is successful", func() {
+			It("generates data, backups up the namespace, deletes the namespace, restores the namespace and verifies data", func() {
+				backupUUID, err := uuid.NewRandom()
+				Expect(err).NotTo(HaveOccurred())
+				backupName = "backup-" + backupUUID.String()
+				restoreName = "restore-" + backupUUID.String()
+				println("backupName = " + backupName)
+				println("creating namespace " + kibishiNamespace)
+				timeoutCTX, _ := context.WithTimeout(context.Background(), time.Minute)
+				err = CreateNamespace(timeoutCTX, kibishiNamespace)
+				Expect(err).NotTo(HaveOccurred())
+
+				println("installing kibishii in namespace " + kibishiNamespace)
+				timeoutCTX, _ = context.WithTimeout(context.Background(), 30*time.Minute)
+				err = InstallKibishii(timeoutCTX, kibishiNamespace, cloudPlatform)
+				Expect(err).NotTo(HaveOccurred())
+
+				println("running kibishii generate")
+				timeoutCTX, _ = context.WithTimeout(context.Background(), time.Minute*60)
+
+				err = GenerateData(timeoutCTX, kibishiNamespace, 2, 10, 10, 1024, 1024, 0, 2)
```

Comment:
It might be nice to wrap the arguments that are needed for kibishii into a struct so that it's a bit clearer what each of these integers represent.